kolibri/content/api.py

```diff
@@ -1,18 +1,23 @@
 from functools import reduce
 from random import sample
 
+import requests
 from django.core.cache import cache
 from django.db.models import Q
 from django.db.models.aggregates import Count
+from django.http import Http404
+from django.utils.translation import ugettext as _
 from kolibri.content import models, serializers
+from kolibri.content.permissions import CanManageContent
+from kolibri.content.utils.paths import get_channel_lookup_url
 from kolibri.logger.models import ContentSessionLog, ContentSummaryLog
 from le_utils.constants import content_kinds
-from rest_framework import filters, pagination, viewsets
+from rest_framework import filters, mixins, pagination, viewsets
 from rest_framework.decorators import detail_route, list_route
 from rest_framework.generics import get_object_or_404
 from rest_framework.response import Response
-from rest_framework import mixins
 from rest_framework.serializers import ValidationError
+from rest_framework.viewsets import ViewSet
```

Comment:
It looks like `rest_framework.viewsets` has been imported in line 15. Do you mind using `viewsets.Viewset` for your class `RemoteChannelViewSet`?

Reply:
ahh yes didn't notice that, good catch!